File: Data Structures & Algorithms/buy-and-sell-crypto/submission-0.py
class Solution:
    def maxProfit(self, prices: List[int]) -> int:
        maxProfit = 0
        mini = prices[0]
        for i in range(1, len(prices)):
            maxProfit = max(maxProfit, prices[i] - mini)
            mini = min(mini, prices[i])
        return maxProfit
    
    
        

File: Data Structures & Algorithms/is-anagram/submission-0.cpp
class Solution {
public:
    bool isAnagram(string s, string t) {
        vector<int> a(26, 0);
        for(auto &x: s)
        {
            a[x - 'a']++;
        }
        for(auto &y: t)
        {
            a[y - 'a']--;
        }

        for(int i = 0; i < a.size(); i++)
        {
            if(a[i] != 0)
            {
                return false;
            }
        }
        return true;
    }
};
